tools/ocft/nfsserver

```diff
@@ -6,8 +6,15 @@ CONFIG
 	InstallPackage nfs-kernel-server
```

Comment:
On Wed, Dec 23, 2015 at 01:26:24AM -0800, oalbrigt wrote:
> > @@ -6,8 +6,15 @@ CONFIG
> >  	InstallPackage nfs-kernel-server
> 
> This package is called nfs-utils on RHEL.

Heh. InstallPackage should support redhat, suse, debian, but what
good is it when the package names differ. Anyway, it turned out
we can do some initialization in SETUP-AGENT. There's now a
new commit here which should make ocft nfsserver work on redhat
too.